code/modules/paperwork/records/info/waw.dm

```diff
@@ -77,15 +106,24 @@
 	Risk Class : Waw	<br>
 	Max PE Boxes : 24	<br>
 	Qliphoth Counter : X	<br>
-	- The first employee who completes their work with a Good result will receive The Knight of Despair�s blessing. (From then on, the employee will be referred to as O-01-73-1.)	<br>
+	Work Damage Type : White	<br>
+	Work Damage : High	<br>
+	- The first employee who completes their work with a Good result will receive The Knight of Despair�s blessing. (From then on, the employee will be referred to as O-01-73-1.)	<br>
 	- O-01-73-1 could only aid in suppressing Abnormalities. Furthermore, thanks to the blessing, RED, WHITE, and BLACK damage dealt to them was halved, however, PALE was doubled.	<br>
-	- The Knight of Despair�s blessing dissipated when O-01-73-1 died. After the blessing was gone, a new employee could receive it.	<br>
+	- The Knight of Despair�s blessing dissipated when O-01-73-1 died. After the blessing was gone, a new employee could receive it.	<br>
```

Comment:
![image](https://user-images.githubusercontent.com/53223414/187028326-4536b423-27c0-48df-bbb8-8e922710f9ca.png)

Reply:
Yeah I had no idea why this was like this? Same for Queenie. I just kinda left it because it was there.